lib/browser/api/app.js

```diff
@@ -77,6 +79,28 @@ app.allowNTLMCredentialsForAllDomains = function (allow) {
   }
 }
 
+const makeSingleInstance = app.makeSingleInstance
```

Comment:
Doing the implement in C++ should be as easy as in JavaScript, the code could be slightly longer but benefits a lot.